client/main.lua

```diff
@@ -2,6 +2,69 @@ local config = require 'config.client'
 local sharedConfig = require 'config.shared'
 local VEHICLES = exports.qbx_core:GetVehiclesByName()
 
+---@enum ProgressColor
+local ProgressColor = {
+    GREEN = 'green.5',
+    YELLOW = 'yellow.5',
+    RED = 'red.5'
+}
+
+---@param percent number
+---@return string
+local function getProgressColor(percent)
+    if percent >= 75 then
+        return ProgressColor.GREEN
+    elseif percent > 25 then
+        return ProgressColor.YELLOW
+    end
+
+    return ProgressColor.RED
```

Comment:
nit; I personally would add the else since the third option is related closely to the other two.

I only use the implied else when using guard clauses. Feel free to ignore this if you don't agree.